src/Sql/Ddl/Column/AbstractPrecisionColumn.php

```diff
@@ -1,86 +1,60 @@
 <?php
 /**
- * Zend Framework (http://framework.zend.com/)
- *
- * @link      http://github.com/zendframework/zf2 for the canonical source repository
- * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
- * @license   http://framework.zend.com/license/new-bsd New BSD License
+ * @see       https://github.com/zendframework/zend-db for the canonical source repository
+ * @copyright Copyright (c) 2005-2019 Zend Technologies USA Inc. (https://www.zend.com)
+ * @license   https://github.com/zendframework/zend-db/blob/master/LICENSE.md New BSD License
  */
 
+declare(strict_types=1);
+
 namespace Zend\Db\Sql\Ddl\Column;
 
 abstract class AbstractPrecisionColumn extends AbstractLengthColumn
 {
-    /**
-     * @var int|null
-     */
+    /** @var int|null */
     protected $decimal;
 
-    /**
-     * {@inheritDoc}
-     *
-     * @param int|null $decimal
-     * @param int      $digits
-     */
     public function __construct(
-        $name,
-        $digits = null,
-        $decimal = null,
-        $nullable = false,
-        $default = null,
+        string $name = '',
```

Comment:
why we have default value here?

Reply:
The original Column.php setName method is a bit confusing. Not sure which way to go here.

Reply:
I can leave the `null` option in the constructor as it was and handle the `null` parameter in the setter to ensure '' (empty string)  
or  
force at least an empty string in the constructor.
Any good ideas here?